scss/application.scss

```diff
@@ -0,0 +1,4 @@
+@import 'equalizer';
+@import 'https://fonts.google.com/specimen/Roboto?selection.family=Roboto';
+@import 'styles';
+@import 'colors';
```

Comment:
add empty line at EOF, `colors` should be imported before `styles` since `styles` could use the variables defined in the `colors` file